src/index.ts

```diff
@@ -88,6 +94,10 @@ export async function resolveSchema (config: Config): Promise<Schema> {
               })
           }
         })
+     case 'schema':
+       const schemaSource = require(resolve(config.file))
```

Comment:
No, "both of them" means with/without `/`. Relative path never begins with `/`. As I said, the common mistake of users is that they use an absolute path instead of relative path. In order to prevent that, we could join the cwd with the given path.